shed/tests/test_streams.py

```diff
@@ -833,6 +833,30 @@ def add(img1, img2):
         source.emit(a)
 
 
+@pytest.mark.xfail(raises=ValueError)
+def test_scan_fail_input_info(exp_db, start_uid1):
+    ''' should fail on bad input info (more than one input).'''
```

Comment:
note this is a fail in the stream setup, not in the actual stream itself (so not in the `stop` document)